src/Microsoft.Azure.SignalR.Common/Utilities/RestClient.cs

```diff
@@ -99,6 +101,8 @@ public Task SendStreamMessageWithRetryAsync(
         return SendAsyncCore(Constants.HttpClientNames.MessageResilient, api, httpMethod, new StreamItemMessage(streamId, arg), typeHint, AsAsync(handleExpectedResponse), cancellationToken);
     }
 
+    public ObjectSerializer ObjectSerializer => _payloadContentBuilder.ObjectSerializer ?? DefaultObjectSerializer;
```

Comment:
I think RestClient should be the one to serialize the http response into InvocationResponse with raw SignalR content, as InvocationResponse is the protocol for REST API only, while the raw SignalR content should be deserialized by maybe user's hub protocol?